sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/config/KeyMgmtConfig.java

```diff
@@ -0,0 +1,32 @@
+package org.sdase.commons.keymgmt.config;
+
+import io.dropwizard.Configuration;
+import javax.validation.constraints.NotNull;
+
+@SuppressWarnings("unused")
+public class KeyMgmtConfig extends Configuration {
```

Comment:
```suggestion
import javax.validation.constraints.NotNull;

@SuppressWarnings("unused")
public class KeyMgmtConfig {
```

We should not extend `Configuration` here.

In [the test](https://github.com/SDA-SE/sda-dropwizard-commons/pull/946/files#diff-1e7c5fa52ea75c40cd57468c1a511dc125eda30157038990c08d9ea7530ea2f9) this class is used as expected as a field type in the configuration.

Reply:
done